widgets/HubspotBatchSource-batchsource.json

```diff
@@ -0,0 +1,129 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "configuration-groups": [
+    {
+      "label": "Basic",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "App ID",
+          "name": "appId"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "API Key",
+          "name": "apiKey"
```

Comment:
is this secret or not? If it is should be password type.

Reply:
it is definitely a secret, the type should be `securekey-text`